samples/Mvc.Client/Mvc.Client.csproj

```diff
@@ -0,0 +1,95 @@
+<Project Sdk="Microsoft.NET.Sdk.Web" ToolsVersion="15.0">
+
+  <PropertyGroup>
+    <RuntimeIdentifiers>win7-x64;win7-x86;osx.10.10-x64;osx.10.11-x64;ubuntu.14.04-x64;ubuntu.16.04-x64;centos.7-x64;rhel.7.2-x64;debian.8-x64;fedora.23-x64;opensuse.13.2-x64</RuntimeIdentifiers>
```

Comment:
@StummeJ Try with ...
```
<RuntimeFrameworkVersion>1.1.0</RuntimeFrameworkVersion>
```
I just found out that sample apps I'm working with will restore and compile for both `netcoreapp1.1` and `net451` if the props are set up this way ...
```
<PropertyGroup>
  <TargetFrameworks>net451;netcoreapp1.1</TargetFrameworks>
  <OutputType>Exe</OutputType>
  <RuntimeFrameworkVersion>1.1.0</RuntimeFrameworkVersion>
</PropertyGroup>
```
```
dotnet restore
dotnet publish -r Release -f netcoreapp1.1
dotnet publish -r Release -f net451
```